html/captive-portal/templates/device-registration/registration.html

```diff
@@ -43,14 +43,18 @@
   <div class="layout__item u-mt u-2/3 u-1/1-palm">
     <a href="/status" class="btn btn--full btn--light">
       <div class="flag">
-        <div class="flag__img">[% svgIcon(id='ic_account_circle_black_24px',size='small') %]</div>
-        <p class="flag__body">[% i18n("Manage registered devices") %]</p>
+        <div class="flag__img">[% svgIcon(id='ic_devices_black_24px',size='small') %]</div>
+        <p class="flag__body">[% i18n("Manage my devices") %]</p>
       </div>
     </a>
-  </div>
-</div>
-<div class="layout layout--center">
-  <div class="layout__item u-mt u-2/3 u-1/1-palm">
+[% IF hasLocalAccount %]
+    <a href="/status/person" class="btn btn--full btn--light">
```

Comment:
Given this is only used to change password, this should be called /status/reset_password or something like this